modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/SilverContextMenu.ts

```diff
@@ -119,15 +120,15 @@ export const isTriggeredByKeyboard = (editor: Editor, e: PointerEvent) =>
 const getSelectedElement = (editor: Editor, e: PointerEvent) =>
   isTriggeredByKeyboard(editor, e) ? editor.selection.getStart(true) : e.target as Element;
 
-const shouldUseNodeAnchor = (editor: Editor, e: PointerEvent) => {
+const getAnchorType = (editor: Editor, e: PointerEvent): AnchorType => {
   const selector = Settings.getAvoidOverlapSelector(editor);
-  if (isTriggeredByKeyboard(editor, e)) {
-    return true;
-  } else if (selector) {
+  const anchorType = isTriggeredByKeyboard(editor, e) ? 'selection' : 'point';
+  if (selector) {
```

Comment:
I realise this was already here, but given we are modifying this code maybe we can use `Strings.isNotEmpty(selector)`?

Reply:
I had done that but reverted cause @lnewson said it was going to be a behaviour change?! Can't remember actually...

Reply:
That was different as you had reordered the ifs previously and that was the behaviour change I mentioned.

Reply:
ah right, changed 👍